port/common/omrsignal.c

```diff
@@ -225,6 +225,51 @@ omrsig_map_portlib_signal_to_os_signal(struct OMRPortLibrary *portLibrary, uint3
 	return OMRPORT_SIG_ERROR;
 }
 
+/**
+ * @brief Register a handler with the OS. For an invalid portlibSignalFlag, an error is returned.
+ * portlibSignalFlag is invalid if it is zero or if multiple signal bits are specified. If
+ * OS fails to register newOSHandler for the specified signal, then an error is returned. In
+ * error cases, oldOSHandler won't be updated.
+ *
+ * This function may override a master handler which was previously set by omrsig_protect or
+ * omrsig_set_*_async_signal_handler variant. The records associated with the master handler
+ * for the portlibSignalFlag signal are left unchanged when this function overrides the master
+ * handler. When the master handler is re-registered with the portlibSignalFlag signal, then
+ * the records associated with master handler don't need to be restored. An example of records
+ * will be J9*AsyncHandlerRecord(s) in asyncHandlerList.
+ *
+ * Each platform variant of omrsignal.c should have a signalMap. signalMap should have a list of
+ * signals, which are supported on a platform. This function will support all signals listed in
+ * omrsignal.c::signalMap.
+ *
+ * On Unix variants, sigaction will be used to register the handler. SA_SIGINFO flag is set.
```

Comment:
Please remove use of "will",  "we", "us".
i.e.
will be -> is
will let -> lets or allows
we cache the original OS handler -> cache the original OS handler
This will allow us to properly restore the original OS handler during shutdown
-> The original OS handler is properly restored during shutdown